### PR TITLE
Inventory Management  Release Notes January 2019

### PR DESCRIPTION
**Pull request by jfrontain**
## This PR is a:

- [ x] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will publish release notes for the MSI module for the first time.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/release-notes/releasenotes-inventory-management.html
- https://devdocs.magento.com/guides/v2.3/inventory/release-notes.html

whatsnew
Added [MSI 1.1.0](https://devdocs.magento.com/guides/v2.3/inventory/release-notes.html) release notes.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3569) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3569) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by lorikrell** [comment]
This content looks great! I have one addition for the notes, feel free to edit as you like! I included the documentation link that will display it.

```- {:.new} Expanded Source Quantity List
Multi Source merchants with a high number of sources can easily hover and view all sources per product through the Product Grid. Each product displays a minimum of five sources and matching quantities. When hovering over the sources, you can scroll through the entire list of sources and current quantities. See [Managing Inventory Quantities](https://docs.magento.com/m2/ce/user_guide/catalog/inventory-manage-inventory-quantities.html).```

**Comment by jeff-matthews** [comment]
@jfrontain, if it would help, I'd be happy to help create the include and push the changes up. Let me know.

**Comment by lorikrell** [comment]
May be helpful to include updates for the DevDocs? 

Updated the Inventory Management [REST tutorial](https://devdocs.magento.com/guides/v2.3/rest/tutorials/inventory/index.html), [REST reference](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/manage-source-selection.html) and [developer information](https://devdocs.magento.com/guides/v2.3/inventory/source-selection-algorithms.html) to include the Distance Priority Source Selection Algorithm.

File: guides/v2.3/release-notes/bk-release-notes.md [hunk 1]
```diff
@@ -11,6 +11,13 @@ title: 2.3.x Release Information
 
 Interested in the Magento 2.1.x and 2.2.x releases? Check out the [2.1.x]({{ site.baseurl }}/guides/v2.1/release-notes/bk-release-notes.html) and [2.2.x]({{ site.baseurl }}/guides/v2.2/release-notes/bk-release-notes.html) Release Information pages.  
 
+
```
Comment (jeff-matthews) [inline guides/v2.3/release-notes/bk-release-notes.md]: We may not need this section.

**Comment by lorikrell** [comment]
1.1.0 MSI metapackage pushed. Good to go for merge. Release approved.

**Comment by lorikrell** [comment]
running tests

**Pull request merged**